shuffle/cli/src/main.rs

```diff
@@ -13,6 +14,10 @@ pub fn main() -> Result<()> {
     match subcommand {
         Subcommand::New { blockchain, path } => new::handle(blockchain, path),
         Subcommand::Node { project_path } => node::handle(project_path.as_path()),
+        Subcommand::Deploy {
+            project_path,
+            key_path,
```

Comment:
Rather than pass in the key path, can we rely on the convention that it will always be in `<project_path>/nodeconfig/mint.key`. 

https://en.wikipedia.org/wiki/Convention_over_configuration

<img width="567" alt="CleanShot 2021-09-24 at 09 35 24@2x" src="https://user-images.githubusercontent.com/635121/134710220-f4316f8a-3957-4a5f-9e0a-40e60846cd47.png">

Reply:
I am using /nodeconfig/mint.key already, the key path is for a user account key pair. This code will be removed entirely when shuffle account is written and deploy will take an account address as a parameter instead

Reply:
This key is different to the mint key, seems to be a new account key? Would make that clearer in naming or comments.